gnome-initial-setup/eos/hack-sound-client/eos-hack-sound-client.c

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Would it be an option to generate this file with gdbus-codegen?

Reply:
(For context, we are planning to eventually make this into an introspectable library)

Reply:
Sounds good to me, but I think this is not feasible for "now" since this has the higher priority now.